x-pack/legacy/plugins/maps/public/shared/layers/vector_layer.js

```diff
@@ -287,6 +287,7 @@ export class VectorLayer extends AbstractLayer {
 
     const searchFilters = {
       ...dataFilters,
+      layerQuery: joinSource.getWhereQuery(),
```

Comment:
is it worth renaming the `layerQuery` property in search-filters to just `filter` now (?) . or `whereFilter` (?)

Reply:
`layerQuery` tag is used in `AbstractESSource._makeSearchSource` so the name has to be the same. I could change this name to something more generic. What do you have in mind? Maybe `sourceQuery`?

Reply:
+1 `sourceQuery`